package.json

```diff
@@ -34,6 +34,7 @@
   },
   "devDependencies": {
     "@reduxjs/toolkit": "^2.2.6",
+    "@sentry/vite-plugin": "^3.2.2",
```

Comment:
why we only move this one to devDependencies ?

Reply:
this fix is for the dependency that requires git. In another PR will introduce the improvement for the update time